packages/mongodb-memory-server-core/src/util/MongoInstance.ts

```diff
@@ -337,29 +336,16 @@ export class MongoInstance extends EventEmitter {
   _launchKiller(parentPid: number, childPid: number): ChildProcess {
     this.debug(`Called MongoInstance._launchKiller(parent: ${parentPid}, child: ${childPid}):`);
     // spawn process which kills itself and mongo process if current process is dead
-    const killer = spawnChild(
-      process.env['NODE'] ?? process.argv[0], // try Environment variable "NODE" before using argv[0]
-      [
-        path.resolve(__dirname, '../../scripts/mongo_killer.js'),
-        parentPid.toString(),
-        childPid.toString(),
-      ],
-      { stdio: 'pipe' }
+    const killer = fork(
+      path.resolve(__dirname, '../../scripts/mongo_killer.js'),
+      [parentPid.toString(), childPid.toString()],
+      {
+        detached: true,
+        stdio: 'ignore', // stdio cannot be done with an detached process cross-systems and without killing the fork on parent termination
```

Comment:
This is sad news.

Do we know what platforms have such problems? Maybe we if it just a win32 then provide killer without stdio support, but for other platforms keep full log/debug functionality.

Or you sure that everything will work without problems and checking stdout/err is redundant?

Reply:
> Do we know what platforms have such problems?

from what i know, its an nodejs quirk, not platform specific

> Or you sure that everything will work without problems and checking stdout/err is redundant?

im actually unsure if even before this change the stdio worked

Reply:
I can suggest a dirty solution for now: just comment the old code (do not remove completely) and keep the new one. This will make it easier to roll back the code in the far future.

Reply:
couldnt we then just revert this commit?

Reply:
fixed in 132917fac63745b351dfd73eb30f7f54962ddf37

we can just revert this commit if we ever need to